docs/usage/administration/request_log.md

```diff
@@ -12,14 +12,14 @@ See the following for how to decode the dense data available from the default lo
 
 | Part  | Explanation | 
 | ----- | ------------ |
-| AAAA  | Timestamp request was logged (not recieved) |
+| AAAA  | Timestamp request was logged (not received) |
 | BBBB  | Logger name (`synapse.access.(http\|https).<tag>`, where 'tag' is defined in the `listeners` config section, normally the port) |
 | CCCC  | Line number in code |
 | DDDD  | Log Level |
 | EEEE  | Request Identifier (This identifier is shared by related log lines)|
 | FFFF  | Source IP (Or X-Forwarded-For if enabled) |
 | GGGG  | Server Port |
-| HHHH  | Federated Server or Local User making request (blank if unauthenticated or not supplied) |
+| HHHH  | Federated Server or Local User making request (blank if unauthenticated or not supplied).<br/>If this is of the form `@aaa:example.com|@bbb:example.com`, then that means that `@aaa:example.com` is authenticated but they are controlling `@bbb:example.com`. |
```

Comment:
Does this only cover an admin puppetting a user via the API, or are there other examples? (Appservices?)

Reply:
Thanks. I'd be tempted to add a snippet says: "e.g. if `aaa` is controlling `bbb` via the admin API" and linking to [the relevant docs](https://matrix-org.github.io/synapse/latest/admin_api/user_admin_api.html#login-as-a-user).

But you might consider that overkill. I'll leave it to your discretion!